remote_write/targets/grafana_agent.go

```diff
@@ -3,12 +3,17 @@ package targets
 import (
 	"fmt"
 	"os"
+
+	"github.com/prometheus/compliance/remote_write/latest"
 )
 
-const grafanaAgentDownloadURL = "https://github.com/grafana/agent/releases/download/v0.13.1/agent-{{.OS}}-{{.Arch}}.zip"
+func getGrafanaDownloadURL() string {
+	version := latest.GetLatestVersion("grafana/agent")
+	return "https://github.com/grafana/agent/releases/download/" + version + "/agent-{{.OS}}-{{.Arch}}.zip"
```

Comment:
Perhaps we could make version a field in the URL template and have `downloadBinary` extract the github project, lookup the latest version and sub it in?

Reply:
I couldn't make it work with templates, but regexp does the job.